airbyte-integrations/connectors/source-linkedin-ads/unit_tests/test_source.py

```diff
@@ -56,7 +56,7 @@
 
 
 class TestAllStreams:
-    _instance: SourceLinkedinAds = SourceLinkedinAds()
+    _instance: SourceLinkedinAds = get_source(TEST_CONFIG)
```

Comment:
I know this was already a problem before but I fear using a class variable because it is common to all instances and therefore could have side effects on other tests. For example, if there is a stateful variable, it will be re-used from tests to test (like the pagination issue we've seen for concurrent). Is it easily fixable?